src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -14,11 +38,18 @@ interface PharmacyProps {
   phone: string;
   website: string;
   lastUpdated: string;
+  vaccineAvailabilities: VaccineAvailabilityTimeslotRequirementExpandedResponse[];
+  vaccineAvailabilitiesNew: VaccineAvailabilitiesByDateAndRequirementsInterface;
 }
 
+// TODO: We don't need to carry both vaccineAvailabilities & vaccineAvailabilitiesNew,
```

Comment:
Get rid of the legacy one, if there are improvements required for the new one we can add spit and polish.